lms/static/sass/discussion/utilities/_variables.scss

```diff
@@ -6,3 +10,8 @@ $forum-color-following: $blue;
 $forum-color-staff: $blue;
 $forum-color-community-ta: $green-d1;
 $forum-color-marked-answer: $green-d1;
+
+// post images
+$post-image-dimension: ($baseline*3);  // image size  + margin
```

Comment:
Ah, so because the variable was in "views/learner-profile" (and not necessarily a base variable / utilitiy) I didn't take that other variable directly, but I could move the learner-profile size out to base/variables perhaps? That may make more sense than what I have here now. 

The benefit of that is we could show all image sizes in the platform in 1 place, and reuse sizes when appropriate. (profile-image, header-image, discussion-post, discussion-response, etc.)